config/default/common/config/wv.json/layers/reference/Reference_Labels_15m.json

```diff
@@ -25,7 +25,7 @@
       },
       "vectorStyle": {
         "id": "Reference_Labels_15m",
-        "url": "https://nasa.maps.arcgis.com/sharing/rest/content/items/b611632010304f9a9358e1eec064cd25/resources/styles/root.json?f=pjson"
+        "url": "https://nasa.maps.arcgis.com/sharing/rest/content/items/8a79d68248f4415fb191077549d982e1/resources/styles/root.json?f=pjson"
```

Comment:
Didn't Ryan just update this the other day? Do we need to update it again for something?

Reply:
Sorry, the base branch was wrong. I changed it to `develop` and I think it's ok now.